packages/dependency-operator-filter/contracts/OperatorFiltererUpgradeable.sol

```diff
@@ -62,4 +64,19 @@ abstract contract OperatorFiltererUpgradeable is Initializable, ContextUpgradeab
         }
         _;
     }
+
+    /// @notice returns the operator filter registry.
+    /// @return address of operator filter registry contract.
+    function getOperatorFilterRegistry() external view returns (IOperatorFilterRegistry) {
+        return _getOperatorFilterRegistry();
+    }
```

Comment:
Why do we do this? I mean having an external function call internal one for a getter?
We could just have one public getter?

Reply:
Following this pattern we could use the internal method to set values when required in the code.

Reply:
This is a get function not a set, so we cannot use it to set anything.

Feels unnecessary to me to have external + internal for a get function when this is exactly when we would use `public` so it can be accessed by both.

Reply:
I'm ok with this